components/modal/modal.css

```diff
@@ -0,0 +1,41 @@
+.modal-container {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 100vw;
+  height: 100vh;
+}
+
+.modal-wrapper {
+  padding: 1rem;
+  width: max-content;
+  background-color: var(--white);
+  display: flex;
```

Comment:
you could create separate class for flex properties which are repeating, or club the classes together to avoid repeating code